src/webhook/mutation/pod_mutator/init_container.go

```diff
@@ -44,34 +43,59 @@ func securityContextForInitContainer(pod *corev1.Pod) *corev1.SecurityContext {
 				"ALL",
 			},
 		},
-		SeccompProfile: &corev1.SeccompProfile{
-			Type: corev1.SeccompProfileTypeRuntimeDefault,
-		},
 	}
 
+	copySecurityContextUserAndGroup(securityContext, *pod)
```

Comment:
it should be more explicit that the security context is modified in this function by either returning the modified security context or only returning user/group and modifying the security context outside the function